WyattHWilliams/react-challenge-project-jan-2022
# #10 Add-Feature: User's Email Display in Nav

## Changes
1. added JSX for users email to display in nav
2. added getting user's email out of auth reducer for display
3. css styling for the email display in nav

## Purpose
addresses [issue 10](https://github.com/Shift3/react-challenge-project-jan-2022/issues/10)

## Approach
we leverage redux to easily update and display the current logged-in user's email.

## Pre-Testing TODOs
1. login the user

## Testing Steps
1. check that the user's email is displayed in the nav component on the viewOrders screen.
2. check that the user's email persists on the nave as the user navigates between pages.

Closes #010
